mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -787,7 +787,8 @@ def LLVM_CallOp : LLVM_MemAccessOpBase<"call",
       OptionalAttr<DictArrayAttr>:$arg_attrs,
       OptionalAttr<DictArrayAttr>:$res_attrs,
       OptionalAttr<UnitAttr>:$no_inline,
-      OptionalAttr<UnitAttr>:$always_inline);
+      OptionalAttr<UnitAttr>:$always_inline,
+      OptionalAttr<UnitAttr>:$inline_hint);
```

Comment:
Can you please document this in the description for the op?

Reply:
Also, `OptionalAttr<UnitAttr>` is redundant unless you *wast* a 3-state boolean. Just `UnitAttr` should do